other/react-native/__tests__/__snapshots__/render-tests.js.snap

```diff
@@ -0,0 +1,315 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I'm normally not a huge fan of huge snapshots ([learn more](https://blog.kentcdodds.com/effective-snapshot-testing-e0d1a2c28eca)). But I think this is ok because I don't think we plan on making changes that would affect this snapshot very often anyway. But if there's a way we can avoid these snapshots by making more explicit assertions I think I'd be happier with that.

Reply:
Agreed. I tried thinking of a better way to get here, instead of using snapshots, but I'm unsure of what other methods we could use to test the React Native renderer.

We could emulate [the Preact tests](https://github.com/paypal/downshift/blob/6f9a306081e960ae707ca3925aee9de834061dda/other/misc-tests/__tests__/preact.js#L24-L44) and use a spy to ensure that `render` successfully gets called, but I don't think that's any better than these snapshots. I'll see if I can come up some better assertions here.

Reply:
I ended up writing some tests that were similar to those Preact tests and I feel like it's much clearer as to what we're testing now! 216c531

Let me know what you think.